frontend/src/routes/about/us/+page.svelte

```diff
@@ -70,58 +58,58 @@
 						)}
 						alt="Profilbild"
 					/>
-					<div class="flex flex-col gap-6 p-4">
-						<h3 class="text-base font-bold">{statement.subject}</h3>
-						<h2 class="text-2xl font-bold uppercase">{statement.name}</h2>
+					<div class="flex flex-col gap-2 p-4 sm:gap-8">
+						<b class="h-4">{statement.subject}</b>
+						<h2>{statement.name}</h2>
 						<p>{statement.statement}</p>
 					</div>
 				</div>
 			{/each}
 		</div>
-	</div>
+	</section>
 
-	<div class="flex flex-col gap-8 px-4 py-20 xl:px-80">
-		<h1 class="break-words text-4xl font-bold uppercase md:text-5xl">
-			Hochschul-SMD <br /> Ein deutschlandweites Netzwerk
-		</h1>
-		<div>
-			Als Hochschulgruppe der SMD sind wir eine ehrenamtliche Initiative. Unterstützt werden wir von
-			einer hauptamtlichen Regionalreferentin und ehrenamtlichen Gruppenbegleitern (SMDler, die ihr
-			Studium schon hinter sich gebracht haben.) Wenn du wissen möchtest, wie die Hochschul-SMD
-			überregional aufgebaut ist, schau dir <a
-				class="text-primary"
-				href="https://www.smd.org/hochschul-smd/hochschul-smd/herzlich-willkommen/"
-			>
-				diese Seite
-			</a> an.
-		</div>
-	</div>
+	<!-- TODO add Leiter Block -->
 
-	<div class="flex flex-col gap-8 px-4 xl:px-80">
-		<h1 class="text-4xl font-bold uppercase md:text-5xl">Leitungsteam</h1>
+	<div class="flex flex-col gap-8 px-4 py-20">
+		<h1 class="text-center">Unser Netzwerk</h1>
+		<div>
+			<img
+				class="pb-6 max-sm:w-full sm:float-right sm:w-[40] sm:max-w-sm xl:max-w-lg"
+				src={regions}
+				alt="Regionen"
+			/>
+			<p class="pt-4">
+				Die SMD ist ein deutschlandweites Netzwerk christlicher Hochschulgruppen. Wir sind eine von
+				ca. 80 anderen SMD-Gruppen in ganz Deutschland. Uns gibt es in fast jeder Univeristätsstadt.
+				Wir lieben es auch uns überregional zu vernetzen und uns auszutauschen. Dafür gibt es zwei
+				Mal im Jahr Veranstaltungen wie das SMD-Forum. Außerdem treffen wir uns auch zu
+				Regionalkonferenzen (RegioKon) innerhalb unserer Region Süd-West (siehe Karte).
 
-		<div class="grid gap-8 lg:grid-cols-2 lg:gap-16">
-			<div class="flex flex-col items-center">
-				<img src={leitung} class="rounded-full" alt="Leitungsteam" />
-				<p class="pt-2">Felix R., Maike, Felix M. (von links nach rechts)</p>
-			</div>
-			<div>
-				Unsere Leitung wird jedes Semester von der Gruppe gewählt. Momentan leiten Felix M., Felix
-				R. und Maike unsere Gruppe. Felix M. ist 23 und studiert Physik, Felix R. ist 21 und
-				studiert Umweltingenieurwesen (Bau) und Maike ist 24 und studiert Grundschullehramt.
-			</div>
+				<a
+					class="flex items-center gap-2"
+					href="https://www.smd.org/hochschul-smd/hochschulgruppen/gruppenliste"
+				>
+					Liste Aller Hochschul-SMD Gruppen
```

Comment:
```suggestion
					Liste aller Hochschul-SMD Gruppen
```
not required, but seems more correct to me